website/content/commands/runner-install.mdx

```diff
@@ -35,6 +35,7 @@ Usage: `waypoint runner install [options]`
 #### docker Options
 
 - `-docker-runner-image=<string>` - The Docker image for the Waypoint runner.
+- `-docker-runner-network=<string>` - The Docker network in which to deploy the Waypoint runner.
```

Comment:
I assume this is unrelated but needed so that docs tests are happy?

Reply:
Yes! 😅 I branched before updating the website docs for my changes upstream.